library/foundation/src/main/java/com/epmedu/animeal/foundation/tabs/model/AnimalType.kt

```diff
@@ -5,5 +5,23 @@ import com.epmedu.animeal.resources.R
 
 enum class AnimalType(@StringRes val title: Int) {
     Dogs(R.string.switch_dogs_title),
-    Cats(R.string.switch_cats_title)
+    Cats(R.string.switch_cats_title);
+
+    // todo it possible to use the enum 'name' instead if the obfuscation will exclude that enum
```

Comment:
fix todo

Reply:
Do you guys have a strategy for how you work with the tech dept?
From my perspective, the TODO means that I understand that there would be improvements in the future.

Maybe a good idea to create a new task with refactoring when the obfuscation will be included?
@VEZE What do you think?
Like // TODO [name task] here we will improve...

Reply:
vote for // TODO [name task] here we will improve...

Reply:
Finally, I decided to remove all TODOs since as I see, in a whole project all enums use the Enum.name values, and I guess in the future, all enums will be excluded from obfuscation.
Thank you guys for the comments!